src/avt/Database/CMakeLists.txt

```diff
@@ -13,50 +13,43 @@
 #
 #****************************************************************************
 
-add_library(avtdatabase_ser)
-if(VISIT_PARALLEL)
-    ADD_PARALLEL_LIBRARY(avtdatabase_par)
-endif()
-
 add_subdirectory(Database)
 add_subdirectory(Formats)
 add_subdirectory(Ghost)
 
-include_directories(
-    ${VISIT_COMMON_INCLUDES}
-    ${VISIT_SOURCE_DIR}/avt/Math
-    ${VISIT_SOURCE_DIR}/avt/VisWindow/VisWindow
-    ${VISIT_SOURCE_DIR}/avt/DBAtts/MetaData
-    ${VISIT_SOURCE_DIR}/avt/DBAtts/SIL
-    ${VISIT_SOURCE_DIR}/avt/MIR/Base
-    ${VISIT_SOURCE_DIR}/avt/MIR/Tet
-    ${VISIT_SOURCE_DIR}/avt/MIR/Youngs
-    ${VISIT_SOURCE_DIR}/avt/MIR/Zoo
-    ${VISIT_SOURCE_DIR}/avt/MIR/Discrete/
-    ${VISIT_SOURCE_DIR}/avt/Pipeline/AbstractFilters
-    ${VISIT_SOURCE_DIR}/avt/Pipeline/Data
-    ${VISIT_SOURCE_DIR}/avt/Pipeline/Pipeline
-    ${VISIT_SOURCE_DIR}/avt/Pipeline/Sinks
-    ${VISIT_SOURCE_DIR}/avt/Pipeline/Sources
-    ${VISIT_SOURCE_DIR}/third_party_builtin/tess2
-    ${VISIT_SOURCE_DIR}/visit_vtk/full
-    ${VISIT_SOURCE_DIR}/visit_vtk/lightweight
-    ${VTK_INCLUDE_DIRS})
-
-# Unless we're doing a VISIT_DBIO_ONLY build then we need avtmir
+set(avtdatabase_ser_DEPENDS
+    visitcommon
+    avtdbatts
+    visit_vtk
+    tess2)
+
+set(avtdatabase_par_SOURCES  ${avtdatabase_ser_SOURCES})
+set(avtdatabase_par_HEADERS  ${avtdatabase_ser_HEADERS})
+set(avtdatabase_par_INCLUDES ${avtdatabase_ser_INCLUDES})
+set(avtdatabase_par_DEPENDS  ${avtdatabase_ser_DEPENDS})
```

Comment:
I think these lines mean that all the `_par` lists are identical to the `_ser` lists because we're setting `_par` versions to the contents of the `_ser` versions. If so, do we really need two separate lists?

Reply:
See my previous comment about variable cleanup done by visit_add_library.